Tweak.x

```diff
@@ -418,7 +418,7 @@ NSDictionary *getIdentifiers() {
 	if (![bundleID isEqualToString:@"com.apple.identityservicesd"])
 		return;
 
-	NSString *filePath = ROOT_PATH_NS(@"%@/.beepserv_wsurl");
+	NSString *filePath = ROOT_PATH_NS(@"/.beepserv_wsurl");
```

Comment:
This is a fair point, but we haven't yet noticed this causing any issues and changing it may break peoples' configs if they've already set up something different at that path. I think we'll just leave it for now unless there start to see permissions issues with that file or something